gensim/scripts/make_wiki.py

```diff
@@ -1 +0,0 @@
-make_wikicorpus.py
```

Comment:
OK, let's do that after fixing the segfault. Compared to the segfault, this is a trivial fix.